.github/workflows/release.yaml

```diff
@@ -0,0 +1,36 @@
+name: Publish NuGet packages
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup dotnet
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: 9.0.x
+
+      - run: dotnet restore
+
+      - name: Build
+        run: dotnet build  --configuration Release --no-restore
```

Comment:
Should this be `dotnet pack` in order to generate the .nupkg file?  Or does `dotnet build` do that also?

Reply:
`dotnet build` will generate the .nupkg file.

For what it's worth, this was (mostly) ripped from nomad-dotnet: https://github.com/fermyon/nomad-dotnet/blob/main/.github/workflows/publish.yaml

Reply:
In any case, I'll switch to `dotnet pack` so we're being more pointed about what we're attempting to package and release. Good call-out.

Reply:
Ah! I recall now. It's because nomad-dotnet declares `<GeneratePackageOnBuild>true</GeneratePackageOnBuild>` in its csproj.